templates/SilverStripe/Admin/Includes/LeftAndMain_NotFound.ss

```diff
@@ -0,0 +1,24 @@
+<div class="cms-content flexbox-area-grow $BaseCSSClasses" data-layout-type="border" data-pjax-fragment="Content">
+
+	<div class="cms-content-header north">
+		<div class="cms-content-header-info vertical-align-items flexbox-area-grow">
+			<div class="breadcrumbs-wrapper">
+				<span class="cms-panel-link crumb last">
+					$ErrorType
+				</span>
+			</div>
+		</div>
+    </div>
+
+	$Tools
```

Comment:
Indentation

Also this template has a mix of 2 spaces and 4 spaces (possibly there's tabs in here?)

Reply:
There were tabs. I've converted those to 4 spaces.